packages/fast-components-react-base/src/horizontal-overflow/README.md

```diff
@@ -1,2 +1,5 @@
 # Horizontal overflow
-The *Horizontal overflow* component accepts items as children. A child can have an optional *slot* property which is either "next" or "previous" which can be used to allow the next/previous click actions.
\ No newline at end of file
+The *horizontal overflow* component accepts items as children. A child can have an optional *slot* property which is either "next" or "previous" which can be used to allow the next/previous click actions.
+
+## Accessibility
+The 'next' and 'previous' buttons are only useful for sighted users, since overflow is a purely visual construct, so they should be hidden from screen readers. If the items in the overflow need to be tabbable, users can add the appropriate event handlers or `tabindex`.
```

Comment:
"If the items in the overflow need to be tabbable, users can add the appropriate event handlers or `tabindex`." 

I think we should either improve this and expand on the documentation, or leave this out. As of now it seems generic and not necessarily useful. What is the scenario where something would need to be tabbable and not already in the tabindex? What assumptions are we making about the implementation where adding event handlers or managing tabindex is the provided solution? This seems more confusing than helpful to me.

Reply:
@janechu for follow-up, as this was based on her direction to add this info.

Reply:
This is based on the native scrolling behavior, if you want items in the horizontal overflow to be tabbed through you have to make it so they can be tabbed to, just as it works when scrolling vertically down a page. It's really just a reminder that since this is based on native scrolling behavior all other rules apply. We could leave it out, as long as we have noted that we are using native scrolling behavior.

Reply:
I think we can note that this is implemented using the native scrolling behavior but leave anything regarding items being "tabbable". To me it doesn't add any value regarding accessibility of this component. Whether it's in the context of native scrolling or not, if you want to tab to something, it either needs to be a focusable element, or you need to manage it's tabindex. I think us partially addressing this adds more confusion than not addressing managing focus on the children of the component.

Reply:
Sure, ok, I'll just include that it is based on native scrolling and that is all.